From 91cb97ceaf53bf4ac0efc72c8d6be1e294908041 Mon Sep 17 00:00:00 2001
From: kiran cr <kirancr13.sr@gmail.com>
Date: Wed, 24 Feb 2021 09:30:25 +0530
Subject: [PATCH 2/2] new line added

---
 Hai.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Hai.txt b/Hai.txt
index a6a5a10..e03182c 100644
--- a/Hai.txt
+++ b/Hai.txt
@@ -1,3 +1,4 @@
 1
 2
-3abc
\ No newline at end of file
+3abc
+one line
\ No newline at end of file
-- 
2.30.0.windows.2

