From 3ee0032e3dfa9b2a567784adab15649b57b38867 Mon Sep 17 00:00:00 2001
From: kiran cr <kirancr13.sr@gmail.com>
Date: Thu, 4 Feb 2021 11:54:58 +0530
Subject: [PATCH 2/5] new line added

---
 Test git1.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Test git1.txt b/Test git1.txt
index 2b72d18..e4f704f 100644
--- a/Test git1.txt	
+++ b/Test git1.txt	
@@ -1,3 +1,4 @@
 hi
 hello
-hai
\ No newline at end of file
+hai
+haii
\ No newline at end of file
-- 
2.30.0.windows.2

