From ba2618a62ba53fb1ee4eef8f7383cc8a85c83d58 Mon Sep 17 00:00:00 2001
From: kiran cr <kirancr13.sr@gmail.com>
Date: Mon, 22 Feb 2021 22:10:57 +0530
Subject: [PATCH 5/5] new line added

---
 Hai.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Hai.txt b/Hai.txt
index 56a6051..7a754f4 100644
--- a/Hai.txt
+++ b/Hai.txt
@@ -1 +1,2 @@
-1
\ No newline at end of file
+1
+2
\ No newline at end of file
-- 
2.30.0.windows.2

